### PR TITLE
: create an 'explorer' plugin from DMT app

### LINKED ISSUE #51: dm-core-packages: extract DMT as a explorer plugin
Be able to specify trough config:
- Work on any entry point (where to start the tree)
- Menu that can turned on/off (view and edit)

### PR DESCRIPTION
**Pull request by soofstad**
## What does this pull request change?
create a new explorer plugin, with same functionality as the DMT app: a tree view where you can right click to open a context menu where different actions can be done.
image

## Issues related to this change
closes #51

### CODE REVIEW

**Comment by KristianKjerstad** [comment]
wait for #59 before ci passes

File: packages/explorer/package.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@development-framework/explorer",
```
Comment (soofstad) [inline packages/explorer/package.json:2]: could we do a clean up on dependencies here?
I suspect there are many unused ones

**Review by soofstad — COMMENTED** [review]
Plugin should not crash if a Dastasource is unavailable

**Review by soofstad — COMMENTED** [review]
Can't approve my own PR.
But; I declare Approved! :+1:

**Pull request merged**